docs/contributing/standards.rst

```diff
@@ -82,18 +82,6 @@ the exception that lines of code may be up to 99 characters long. Docstrings and
 tests are required for all public methods, although the flake8_ tool used by
 Deis does not enforce this.
 
-
-Design Proposals
```

Comment:
I'd add a link back in this doc to the new design proposal document, as you did with the contributing overview.